wsgate/pluginTemplate/pluginCommon.cpp

```diff
@@ -8,7 +8,13 @@ void split(std::string input, std::vector<std::string>& tokens, char delim){
     }
 }
 
-bool EXPORT_FUNC queryPlugin(std::string queryInput, std::map<std::string, std::string> & result){
+#ifdef _WIN32
+bool EXPORT_FUNC
```

Comment:
on linux why not just:
    #define EXPORT_FUNC

So we can avoid all those #ifdefs?